instagram/src/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import './App.css';
+import dummyData from './dummy-data';
+import SearchBar from './components/SearchBar/SearchBar.js';
+import PostContainer from './components/PostContainer/PostContainer.js';
+import PostsPage from './components/PostContainer/PostsPage.js';
+import Login from './components/Login/Login.js';
+import Authenticate from './components/Authentication/Authenticate.js';
+
+
+class App extends Component {
+  constructor() {
+    super();
```

Comment:
Similarly - `super()` calls the constructor of its parent class. This is required when you need to access some variables from the parent class. There is no variables that you are trying to access.